### PR TITLE
GLTFExporter Doc: Update for new options

### PR DESCRIPTION
**Pull request by takahirox**
This PR updates `GLTFExporter` doc for new options and fixes a minor stuff.

### CODE REVIEW

File: docs/examples/exporters/GLTFExporter.html [hunk 1]
```diff
@@ -80,7 +80,9 @@ <h3>[method:null parse]( [param:Object input], [param:Function onCompleted], [pa
 			<li>truncateDrawRange - bool. Export just the attributes within the drawRange, if defined, instead of exporting the whole array. Default is true.</li>
 			<li>binary - bool. Export in binary (.glb) format, returning an ArrayBuffer. Default is false.</li>
 			<li>embedImages - bool. Export with images embedded into the glTF asset. Default is true.</li>
-			<li>animations - Array<[page:AnimationClip AnimationClip]>. List of animations to be included in the export.
+			<li>animations - Array<[page:AnimationClip AnimationClip]>. List of animations to be included in the export.</li>
+			<li>forceIndices - bool. Generate indices for non-index geometry and export with it. Default is false.</li>
```
Comment (fernandojsg) [inline docs/examples/exporters/GLTFExporter.html]: Not sure about the wording here, maybe "with them"? Or `Export non-indexed geometry by creating temporary indices for it`? /cc @mrdoob @donmccurdy

**Comment by mrdoob** [comment]
I don't think I would document this these. As they're just temporal workarounds for Facebook 3D posts...?

File: docs/examples/exporters/GLTFExporter.html (hunk 1 above)
Comment (takahirox) [reply docs/examples/exporters/GLTFExporter.html]: Thanks, Replaced "it" with "them" so far.

**Comment by takahirox** [comment]
Will we remove them once FB fixes the limitations?

**Comment by mrdoob** [comment]
I hope so. At least the `forceIndices` one, as that's basically a hack.

**Comment by fernandojsg** [comment]
I believe `forcePOT` could stay, because some engines, specially on mobile maybe could still have that limitation.
And regarding non-indexed to indexed it's true that it won't be useful probably after facebook will implement it, but I would just keep it documented until we'll remove from the code, and once we'll remove from the code, remove also the documentation for the sake of consistency

**Comment by takahirox** [comment]
I wanna add note "These're temporal workarounds for FB" to Doc (or in the code) rather than non documenting.

**Comment by mrdoob** [comment]
Okay, lets remove when they're not needed then.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**